practice_project/isPrime.py

```diff
@@ -0,0 +1,17 @@
+def is_prime(number) :
+    if number <= 1:
+        return False
+    elif number <= 3:
+        return True
+    
+    i = 2
+    while i * i <= number: # check the divisibility of each integer up to the square root of the number
+         if number % i == 0:
+              return False
+         i += 1
+    return True
+         
+if __name__ == '__main__':
```

Comment:
nit: Since you have a test file that basically does the same thing, I would just remove the `main` block here!